generators/app/index.js

```diff
@@ -238,6 +238,7 @@ module.exports = class JHipsterAppGenerator extends BaseBlueprintGenerator {
 
         useBlueprints = !this.fromBlueprint && this.instantiateBlueprints('app');
 
+        this.registerGeneratedAnnotationTransform();
```

Comment:
Following https://github.com/jhipster/generator-jhipster/pull/12569#issuecomment-697232896
```suggestion
        if (this.options.skipGeneratedByJhipster) {
            this.registerGeneratedAnnotationTransform();
        }
```
And add to `this.option('skip-generated-by-jhipster'...`